example-apps/internal-knowledge-search/app-ui/.env

```diff
@@ -1,3 +1,4 @@
-SEARCH_APP_NAME=some-search-application
-SEARCH_APP_API_KEY=xxxxxxxxxxxxxxxxxxx
-SEARCH_APP_ENDPOINT=https://some-search-end-point.co
\ No newline at end of file
+REACT_APP_SEARCH_APP_NAME=changeme
```

Comment:
These are on purpose. This file has 2 over-arching changes:
1. changing the prefix of the variables to be `REACT_` which is apparantly necessary. Lio has made this same change in an isolated PR: https://github.com/elastic/elasticsearch-labs/pull/142
2. removing the API config to replace it with a username/password combo, since API keys cannot be used to create other API keys. See [this slack thread](https://elastic.slack.com/archives/C0D8ST60Y/p1703017952223129).